.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow will install Python dependencies, run the Flake8 python linter (uses Pep8 standards)
+# and run tests with a variety of Python versions
+name: continuous_integrations
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, windows-latest, macos-latest ]
+        python-version: ["3.8", "3.9", "3.10"]
```

Comment:
The SRS doc specifies that we will only be using a linux environment. We could probably remove the MacOS and Windows build matrices. Additionally, the PyPy setup file requires that python 3.9 or later is used.

Reply:
Sounds good thank you!

Reply:
Resolved in d76010fa0722d00c4db5803b5aa6afcd52c18f2e